docs/E-Secondary-Storage/records-and-files.md

```diff
@@ -19,9 +22,9 @@ This chapter describes how to identify records and fields in a text file and how
 
 A **record** occupies a single line in a text file and holds all of the data associated with one chunk of information.  The record is a sequence of characters that ends with a record delimiter.  The typical record delimiter is the newline character (`\n`). 
 
```

Comment:
Hi, there! There is a small issue that is easy to skip for us, which is these auto generated md files would break sentences with 2 spaces `  `  instead of just 1 space ` ` . I think you can make that as an improvement for you PR if you want. Because it's looking good at real page, but it looks a little bit weird in the md file.

Reply:
ok I will fix the extra spaces.